addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -402,6 +402,7 @@ var ListRenderer = BasicRenderer.extend({
             data: record.data,
             escape: true,
             isPassword: 'password' in node.attrs,
+            digits: node.attrs.digits && JSON.parse(node.attrs.digits),
```

Comment:
So in tree view, it does not take into account digits?

Reply:
@jco-odoo yes, and changes are done as suggested

Reply:
@ged-odoo @VincentSchippefilt Could you live with such a change?  We see that digits= in the XML view is interpreted in the form view, but not in the tree view.  (or could there be a reason for this asymmetry?)

Reply:
looks good, thanks for the test :)